.ruby-version

```diff
@@ -1 +1 @@
-2.3.1
+ruby-2.4.1
```

Comment:
Yes, rbenv also supports the use of `.ruby-version`.  Does the presence of the "ruby-" string break rbenv? (if you use rbenv in your local env could you check that?)